apis/validator/aggregate_attestation.v2.yaml

```diff
@@ -0,0 +1,59 @@
+get:
+  operationId: "getAggregatedAttestationV2"
+  summary: "Get aggregated attestation"
+  description: |
+    Aggregates all attestations matching given attestation data root, slot and committee index.
+
+    A 503 error must be returned if the block identified by the response
+    `beacon_block_root` is optimistic (i.e. the aggregated attestation attests
+    to a block that has not been fully verified by an execution engine).
+
+    A 404 error must be returned if no attestation is available for the requested
+    `attestation_data_root`.
+  tags:
+    - ValidatorRequiredApi
+    - Validator
+  parameters:
+    - name: attestation_data_root
+      in: query
+      required: true
+      description: "HashTreeRoot of AttestationData that validator wants aggregated"
+      schema:
+        $ref: '../../beacon-node-oapi.yaml#/components/schemas/Root'
+    - name: slot
+      in: query
+      required: true
+      schema:
+        $ref: '../../beacon-node-oapi.yaml#/components/schemas/Uint64'
+    - name: committee_index
+      in: query
+      required: true
+      schema:
+        $ref: '../../beacon-node-oapi.yaml#/components/schemas/Uint64'
+  responses:
+    "200":
+      description: "Returns aggregated `Attestation` object with same `AttestationData` root, slot and committee index."
```

Comment:
Noticed the description will not be correct post-electra as `index` in `AttestationData` will be set to 0 and not match the  value passed in `committee_index` query param.

Reply:
@nflaig good catch. Will update now

Reply:
> Noticed the description will not be correct post-electra as `index` in `AttestationData` will be set to 0 and not match the value passed in `committee_index` query param.

I think we can interpret this as:
```
Returns aggregated `Attestation` object with the same root and slot in `Attestation.data`, and same committee index in `Attestation`. 
```

I think this description still holds true. Just that the committee index is now in `Attestation` and not `AttestationData`